spring-cloud-sleuth-stream/pom.xml

```diff
@@ -59,6 +58,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.aspectj</groupId>
```

Comment:
In order to use spring-cloud-commons' InetUtils as bean, I added UtilAutoConfiguration to spring context in tests. For some reason aspectj is needed then.